soco/core.py

```diff
@@ -134,6 +135,7 @@ class SoCo(_SocoSingletonBase):
         get_composers -- Get composers from the music library
         get_tracks -- Get tracks from the music library
         get_playlists -- Get playlists from the music library
+        get_sonos_playlists -- Get Sonos playlists from the music library.
```

Comment:
It wasn't originally listed, so I added it to the main docstring.  I've
modified it now say:
  get_sonos_playlists -- Get Sonos playlists.

Too little or just right?

On Mon, Sep 7, 2015 at 10:44 AM, Lawrence Akka <notifications@github.com>
wrote:

> In soco/core.py
> <https://github.com/SoCo/SoCo/pull/352#discussion_r38870472>:
>
> > @@ -134,6 +135,7 @@ class SoCo(_SocoSingletonBase):
> >          get_composers -- Get composers from the music library
> >          get_tracks -- Get tracks from the music library
> >          get_playlists -- Get playlists from the music library
> > +        get_sonos_playlists -- Get Sonos playlists from the music library.
>
> It is not getting playlists from the music library (which, strictly, is
> the locally stored music, i.e. not Spotify etc).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SoCo/SoCo/pull/352/files#r38870472>.
>



-- 
Best,

Jeff Hinrichs
dundeemt@gmail.com
402.218.1473